supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchAsyncBackendJobExecutionActor.scala

```diff
@@ -162,6 +185,10 @@ class AwsBatchAsyncBackendJobExecutionActor(override val standardParams: Standar
       jobPaths, Seq.empty[AwsBatchParameter],
       configuration.awsConfig.region,
       Option(configuration.awsAuth))
+
+    Log.info(s"${this.getClass.getName} initialized AwsBatchJob: ${job.toString}")
```

Comment:
It writes to a single line in the server log (albeit a long line). I have found it to be super useful for diagnosing many issues as it will reveal lots of information about which backend nodes, containers, clusters etc are involved for any given job (something which users have asked for before). It could go into a `DEBUG` but cromwell in `DEBUG` mode seems to produce insane amounts of logging.